File: snippets/python/bubble_sort.py
def bubble_sort(array):
	for _ in range(len(array)-1):
		sorted = True
		for i in range(len(array)-1):
			if array[i] > array[i+1]:
				temp = array[i+1]
				array[i+1] = array[i]
				array[i] = temp
				sorted = False
		if sorted:
			return array

	return array

File: snippets/javascript/bubbleSort.js
function bubble_sort(array) {
	for (i = 0; i < array.length - 1; i++) {
		let sorted = true;
		for (j = 0; j < array.length - 1; j++) {
			if (array[j] > array[j + 1]) {
				let temp = array[j + 1];
				array[j + 1] = array[j];
				array[j] = temp;
				sorted = false;
			}
		}
		if (sorted) return array;
	}
	return array;
}